low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1228,6 +1334,15 @@ def _parse_voltage_info(self, index, date, _time, sensor, sensor_num, event, sta
             "description": event
         }
 
+        if (threshold.lower() in ['low', 'high'] and not status.lower == "deasserted"):
+            self.faulty_resources[sensor_name] = {
+                'status': status,
+                'event': event,
+                'resource': 'voltage'
```

Comment:
This should be corrected as mentioned for other fru types.
Please take care.

Reply:
Done